lib/minitest/mock.rb

```diff
@@ -169,6 +169,8 @@ def method_missing sym, *args, &block # :nodoc:
       retval
     end
 
+    ruby2_keywords :method_missing if respond_to?(:ruby2_keywords, true)
```

Comment:
Unfortunately while this is a nice small diff, it doesn't really cover what needs to be done for kwargs to be fully supported. I think I've got that in place now, but I'd love your eyes on my changes to see if I missed anything.